app/src/main/java/io/pslab/activity/RoboticArmActivity.java

```diff
@@ -339,8 +358,14 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
                 removeStatusBar();
                 if (actionId == EditorInfo.IME_ACTION_DONE) {
                     degree = Integer.valueOf(degreeText4.getText().toString());
-                    seekArc4.setProgress((int) (degree / 3.6));
-                    editEnter = true;
+                    if (degree > 360 || degree < 0) {
+                        degreeText4.setText(getResources().getString(R.string.zero));
+                        seekArc4.setProgress(0);
```

Comment:
It does not impact the working but degreeText1, 2, 3 ,4 should be used respectively , right now there is degreeText4 for all 4 servo layouts.

Reply:
done. Thanks for pointing out